script/Ciel/Day 10/10_01_CIEL10_5B.txt

```diff
@@ -241,12 +261,12 @@ I realize that I have, right now, with my own hands, brought down the curtain on
 [sha:848dfeae88ab7d3ae0ad489fc6fde5b194d77d97]{
 -- Page 372, Offset 28077.
 -- それは確かな事だと、俺も思います」
-I believe this is true."
+I don't doubt that she was telling the truth."
 }
 [sha:3b80866b918b9864151d7310280d99ddbedc0614]{
 -- Page 372, Offset 28078.
 -- 「……アルクェイドの発言は、信じられません」
-"...I can't trust Arcueid's word."
+"I can't just take Arcueid's word for it."
```

Comment:
"I can't just take Arcueid's word for it" makes it pretty apparent what she means, in which case there wouldn't be the need of clarification. It doesn't lead into the next line well, nor does it fit well with her clarification later.
>I don't believe what Arcueid is saying.